### PR TITLE
Sync CLI options with SDK options

### LINKED ISSUE #123: Resolve `rustup default stable` DX
Right now developer's have to run `rustup default stable` to add underlying rust-related CLI dependencies, whereafter that's cached inside of `.hermit/`. But this is less than ideal; hermit should bring these matters out of the box.

```
rs on  kendall/justfile [$] via 🦀 1.77.2
➜ rm -rf .hermit

rs on  kendall/justfile [$] via 🦀  took 20.1s
➜ cargo
error: rustup could not choose a version of cargo to run, because one wasn't specified explicitly, and no default is configured.
help: run 'rustup default stable' to download the latest stable release of Rust and set it as your default toolchain.

rs on  kendall/justfile [$] via 🦀
➜ rustup default stable
info: syncing channel updates for 'stable-aarch64-apple-darwin'
info: latest update on 2024-04-09, rust version 1.77.2 (25ef9e3d8 2024-04-09)
info: downloading component 'cargo'
  6.2 MiB /   6.2 MiB (100 %)   6.0 MiB/s in  1s ETA:  0s
info: downloading component 'clippy'
info: downloading component 'rust-docs'
 14.9 MiB /  14.9 MiB (100 %)   4.8 MiB/s in  3s ETA:  0s
info: downloading component 'rust-std'
 22.9 MiB /  22.9 MiB (100 %)   8.2 MiB/s in  2s ETA:  0s
info: downloading component 'rustc'
 54.1 MiB /  54.1 MiB (100 %)   8.4 MiB/s in  6s ETA:  0s
info: downloading component 'rustfmt'
info: installing component 'cargo'
info: installing component 'clippy'
info: installing component 'rust-docs'
 14.9 MiB /  14.9 MiB (100 %)   3.5 MiB/s in  2s ETA:  0s
info: installing component 'rust-std'
 22.9 MiB /  22.9 MiB (100 %)  19.7 MiB/s in  1s ETA:  0s
info: installing component 'rustc'
 54.1 MiB /  54.1 MiB (100 %)  20.3 MiB/s in  2s ETA:  0s
info: installing component 'rustfmt'
info: default toolchain set to 'stable-aarch64-apple-darwin'

  stable-aarch64-apple-darwin installed - rustc 1.77.2 (25ef9e3d8 2024-04-09)


rs on  kendall/justfile [$] via 🦀 1.77.2 took 26.9s
➜ cargo
Rust's package manager

Usage: cargo [+toolchain] [OPTIONS] [COMMAND]
       cargo [+toolchain] [OPTIONS] -Zscript <MANIFEST_RS> [ARGS]...
```

### PR DESCRIPTION
**Pull request by andresuribe87**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [x] 🚩 Other

## Description

This PR allows the web5 cli to pass in a number of options when creating a VC. 

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### No tests? Add a note

This is simple plumbing

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [x] 📓 general documentation
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: crates/web5_cli/src/vcs.rs [hunk 1]
```diff
@@ -92,6 +121,24 @@ impl Commands {
                         }
                     },
                 };
+                let issuance_date = match issuance_date {
```
Comment (nitro-neal) [inline crates/web5_cli/src/vcs.rs:124]: can issuance date be none?

File: crates/web5_cli/src/vcs.rs [hunk 2]
```diff
@@ -92,6 +121,24 @@ impl Commands {
...
+                    None => None,
```
Comment (nitro-neal) [inline crates/web5_cli/src/vcs.rs:125]: Here it should do the issuance date now()?

**Review by nitro-neal — APPROVED** [review]
Also do you mind switching it to use chrono instead of System time for generating the timestamp:

There was unknown wasm black hole when using it
here was a fix in tbdex-rs - https://github.com/TBD54566975/tbdex-rs/commit/6bac761d43dba9aa4cf855b6e1bb612ad70c5203

```
use chrono::{Duration, Utc};
use std::time::SystemTime;

let now = Utc::now();
let exp = now + Duration::seconds(60);

let now_system_time: SystemTime = now.into();
let exp_system_time: SystemTime = exp.into();
```

File: crates/web5_cli/src/vcs.rs (hunk 2 above)
Comment (andresuribe87) [reply crates/web5_cli/src/vcs.rs:127]: The SDK will take care of that. This is just meant to be a passthrough from the CLI to the SDK function.

**Pull request merged**